### ISSUE TITLE
Next Price order: PositionCard fills before order is executed

### ISSUE THREAD

**Issue by platschi** [Kwenta/kwenta#1009]
### Subject of the issue
When opening a next price order, the `PositionCard` already fills with data before the order is executed. We should fill in the `PositionCard` data only once the Open Order is executed by a Keeper.

Also when closing an order with next-price, the data in the `PositionCard` already moves uPnL to rPnL while the next-price closing order is still waiting to be executed.

### Deliverables
- Avoid filling `PositionCard` with data before a next price order is executed

**Comment by Tburm** [comment]
@platschi I'm not exactly sure how to solve this problem. Potential solutions:
1. We optimistically fill the position card with data from the filled order
2. We avoid refetching/refilling position card data when a next price order is open

(1) is not a good solution because we don't know the price of their fill, and in the case of a failed transaction we would need to revert the data. This would be a bad UX since there is no guarantee the order is filled, and no good way to predict the price.

(2) Is better, but need to figure out the edge cases. We can delay updating on the frontend, however the contract data will update regardless. In the case that the user cancels an order, they "realize" any pnl at the time they submitted the order and the time they cancelled it. There are case where if we "hide" or delay the update, we still need to show it eventually.

**Comment by platschi** [comment]
Given its difficult to obscure information for now this is marked done as wontfix. Might be reopened again if we reorganize how we deal with `unrealized/realized` PnL displays on the frontend.
